custom-fonts.md

```diff
@@ -23,6 +41,72 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Using the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+as a constant. 
+Then set the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html) `fontFamily` property 
+and use the font family specified in the `pubsec.yaml` delcarations. 
+The font family `pubsec.yaml` declaration will look something like `family: [declared-font-family]`.
+
+```
+// declare the text style
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+);
+
+// use the text style
+var buttonText = const Text(
+  "Use the font for this text",
+  style: textStyle,
+);
+```
+
+## Using the included Material Design fonts
+
+When you want to use the material design fonts,
+they can be simply included by adding a property `uses-material-design: true` to the `pubspec.yaml` file.
+
+```
+flutter:
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the Icons class.
+  uses-material-design: true
+```
+
+### Included Material Design fonts
+
+Here are the included fonts that come with `uses-material-design: true`.
+
+| Font | Features |
+| ---- | -------- |
+| Roboto-Regular.ttf | |
+| Roboto-Italic.ttf | italic |
+| Roboto-Thin.ttf | 100 |
+| Roboto-ThinItalic.ttf | 100, italic |
+| Roboto-Light.ttf | 300 |
+| Roboto-LightItalic.ttf | 300, italic |
+| Roboto-Medium.ttf | 500 |
+| Roboto-MediumItalic.ttf | 500, italic |
+| Roboto-Bold.ttf | 700 |
+| Roboto-BoldItalic.ttf | 700, italic |
+| Roboto-Black.ttf | 900 |
+| Roboto-BlackItalic.ttf | 900, italic |
+| &nbsp; | &nbsp; |
+| RobotoCondensed-Regular.ttf | |
+| RobotoCondensed-Italic.ttf | italic |
+| RobotoCondensed-Light.ttf | 300 |
+| RobotoCondensed-LightItalic.ttf | 300, italic |
+| RobotoCondensed-Bold.ttf | 700 |
+| RobotoCondensed-BoldItalic.ttf | 700, italic |
+
+The included fonts
```

Comment:
This is an incomplete sentence :)

Maybe "Check out the material_fonts.yaml to view all the included fonts."